.github/workflows/tests_and_reports.yml

```diff
@@ -126,59 +123,73 @@ jobs:
           path: bundle-visualizer
           retention-days: 1
 
-  deploy-reports-to-gh-pages:
-    name: "Deploy Reports to GitHub Pages"
+  deploy-reports-to-vercel-meta-site:
+    name: "Deploy Reports to Vercel Meta-Site"
     # Wait for tests to complete before deploying.
     needs: [lint-and-unit-tests, end-to-end-tests, bundle-visualizer]
     # We still want to upload reports regardless of whether tests failed.
     if: ${{ always() }}
     runs-on: ubuntu-latest
+    env:
+      VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+      VERCEL_PROJECT_ID: ${{ secrets.VERCEL_META_SITE_PROJECT_ID }}
```

Comment:
These env vars just need to be set; the Vercel CLI will automatically pick these up, so we don't need to provide them as flags (but for some reason we do need to provide `VERCEL_TOKEN` each time).